hash.c

```diff
@@ -3234,12 +3256,11 @@ rb_hash_size_num(VALUE hash)
 
 /*
  *  call-seq:
- *     hsh.empty?    -> true or false
- *
- *  Returns <code>true</code> if <i>hsh</i> contains no key-value pairs.
- *
- *     {}.empty?   #=> true
+ *    hash.empty? → true or false
```

Comment:
This arrow appears to be unicode, is that okay? In other places it was `->`.

Reply:
Fixed.